schema/project-level/project-schema.json

```diff
@@ -1200,7 +1200,7 @@
         },
         "documentType": {
           "title": "Document type",
-          "description": "A classification of the document described taken from the [documentType codelist](http://standard.open-contracting.org/1.1/en/schema/codelists/#document-type). Values from the provided codelist should be used wherever possible, though extended values can be provided if the codelist does not have a relevant code.",
+          "description": "A classification of the document described taken from the [documentType codelist](http://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/projects/reference/#document-type). Values from the provided codelist should be used wherever possible, though extended values can be provided if the codelist does not have a relevant code.",
```

Comment:
I think there would be no hyphen in the anchor (`#document-type`), if the heading is constructed the same as the others.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/101/commits/a1cbe03fd9f1373507d68f440cc4ed8db5e765bf